pkg/containerruntime/config.go

```diff
@@ -45,27 +46,38 @@ func BuildConfig(opts Opts) (Config, error) {
 		}
 	}
 
-	var registryMirrors []string
+	// we want to match e.g. docker.io=registry.docker-cn.com, having docker.io as the first
```

Comment:
I initially worked with `:` instead if `=`, so `docker.io:registry.docker-cn.com`, which looks a bit nicer to me - but there are already flags with `Key=Value` so matching those is probably best?